backend/packages/Upgrade/.env.docker.local.example

```diff
@@ -57,6 +57,7 @@ SWAGGER_JSON=/swagger.json
 # Google Client Id
 #
 GOOGLE_CLIENT_ID=google_project_id
```

Comment:
maybe a comment in the .example files that it can be a single value or a comma separated list (no spaces)

Reply:
we'll need to remember update the gitbook also